README.md

```diff
@@ -595,6 +595,32 @@ Untracked files:
 nothing added to commit but untracked files present (use "git add" to track)
 ```
 
+### masterAdm
```

Comment:
I have a few things in mind, but wanted to get started, hence the name of this command. Other things:
- Delete a range or ranges of exercises
- Create a gap of a given size @ a certain exercise number
- Create a gap @ a certain exercise and duplicate the code from previous or next exercise.
- ...